packages/playground/vite.config.ts

```diff
@@ -5,17 +5,20 @@ import { fileURLToPath } from 'url';
 import { hmrPlugin } from './scripts/hmr-plugin';
 import istanbul from 'vite-plugin-istanbul';
 
+const enableIstanbul = !!process.env.CI || !!process.env.COVERAGE;
+
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [
     react(),
     hmrPlugin,
-    istanbul({
-      cwd: fileURLToPath(new URL('../..', import.meta.url)),
-      include: ['packages/**/src/*'],
-      exclude: ['node_modules', 'tests'],
-      forceBuildInstrument: !!process.env.CI || !!process.env.COVERAGE,
-    }),
+    enableIstanbul &&
```

Comment:
Will this pass in a `false` to `plugins` array?

Reply:
yeah, I think vite config can accept false values in `plugins` field.